actions/command.yaml

```diff
@@ -53,7 +53,7 @@ parameters:
     description: "Connection type to use (default=smart) [-c]"
     type: string
   extra_vars:
-    description: "List of additional variables to pass to ansible. Each variable is represented as \"key=value\" or \"@path/to/file.yaml|json\" [-e]."
+    description: "List of additional variables to pass to ansible. Each variable is represented as \"key=value\" or \"@path/to/file.yaml|json\" or use a yaml dict (\"key: [1, 2, 3]\") to send JSON. [-e]"
```

Comment:
Could be just replaced with something like `Description: '..."key=value"...'` to avoid escaping double quotes.